.circleci/config.yml

```diff
@@ -1,25 +0,0 @@
-version: 2.1
```

Comment:
Does this mean bye bye for circle for this project!?

Reply:
Yep, indeed!